### PR TITLE
Create Matt,_Royce_and_Kevin_Teamwork_Dream_Work.ipynb

### PR DESCRIPTION
**Pull request by LynnerKevin**
## Description

After discussion and group programing with Matthew Clibanoff and Royce Roberts, I added some additional steps to their existing matching system to make it a little smarter as a blueprint for additional layers later.

[Loom](https://www.loom.com/share/3d4c5014670a44b49c0f6d7ce0fb465fl)

## Type of change

- [x] New feature

## Checklist:

- [x] My code follows the PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by mattclibanoff — COMMENTED** [review]
I love the idea to add overlap as a feature along with the cosine similarity. I think it's a great first step towards weighting the similarities we want to focus one most!

**Review by sysgear — APPROVED** [review]
overlap as a feature along with the cosine similarity is a good idea and agreed

**Review by jinjahninjah — APPROVED** [review]
I agree with Matt about the overlap column. I also liked how you made a preen method that didn't work the way you wanted, gives insight into your process. Awesome Job Kevin! 

I also changed the 1s in your description to "x"s to make the markdown work. Also, great video!

File: notebooks/Matt,_Royce_and_Kevin_Teamwork_Dream_Work.ipynb [hunk 1]
```diff
@@ -0,0 +1,2898 @@
...
+      "execution_count": 3,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "mentor = Mentor()\n",
+        "\n",
+        "mentor_df = mentor.to_df(40)\n",
+        "mentor_df['key']=1 \n"
```
Comment (Cole-Richards) [inline notebooks/Matt,_Royce_and_Kevin_Teamwork_Dream_Work.ipynb]: Unclear as to what the key column does here.

**Review by Cole-Richards — APPROVED** [review]
The improvements made look to contribute to the overall conversation regarding how matching will occur and what model will be used. I particularly like the clean_data function.

**Review by Katerynapass — APPROVED** [review]
I thing it is great idea to add overlap as a feature along with the cosine similarity. Great job!

**Review by dankositzke — APPROVED** [review]
Cool upgrade to the recommendations. I didn't know how to sort using two columns so thanks for teaching me something new!

**Comment by nhockcuncon77** [comment]
Impressive work

**Review by nhockcuncon77 — APPROVED** [review]
This is impressive work and I love how you approach that.

**Pull request merged**